src/libzippp.h

```diff
@@ -128,14 +128,22 @@ namespace libzippp {
             CURRENT
         };
         
+        enum Encryption{
+            NONE,// No encryption.
+            AES_128,// Winzip AES-128 encryption.
+            AES_192,// Winzip AES-192 encryption.
+            AES_256,// Winzip AES-256 encryption.
+            TRAD_PKWARE// Traditional PKWare encryption. Do not use this method, it is not secure. It is only provided for backwards compatibility.
+        };
```

Comment:
Please add a description with `/** */` on top of the enum with also describe each of the option (have a look at `OpenMode` for instance).